[2025-08-22T06:32:49.509+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-22T06:32:49.556+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: stock_etl_pipeline.extract__2 scheduled__2025-08-21T00:00:00+00:00 [queued]>
[2025-08-22T06:32:49.565+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: stock_etl_pipeline.extract__2 scheduled__2025-08-21T00:00:00+00:00 [queued]>
[2025-08-22T06:32:49.565+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 3
[2025-08-22T06:32:49.580+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): extract__2> on 2025-08-21 00:00:00+00:00
[2025-08-22T06:32:49.585+0000] {standard_task_runner.py:63} INFO - Started process 243 to run task
[2025-08-22T06:32:49.592+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'stock_etl_pipeline', 'extract__2', 'scheduled__2025-08-21T00:00:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/fetch_stock_prices.py', '--cfg-path', '/tmp/tmpxoarz6p1']
[2025-08-22T06:32:49.594+0000] {standard_task_runner.py:91} INFO - Job 6: Subtask extract__2
[2025-08-22T06:32:49.699+0000] {task_command.py:426} INFO - Running <TaskInstance: stock_etl_pipeline.extract__2 scheduled__2025-08-21T00:00:00+00:00 [running]> on host 25f32958f358
[2025-08-22T06:32:49.853+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='stock_etl_pipeline' AIRFLOW_CTX_TASK_ID='extract__2' AIRFLOW_CTX_EXECUTION_DATE='2025-08-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-08-21T00:00:00+00:00'
[2025-08-22T06:32:49.855+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-22T06:32:49.857+0000] {fetch_stock_prices.py:55} INFO - Extracting data for symbol: GOOGL
[2025-08-22T06:32:51.070+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-22T06:32:51.082+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 265, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/fetch_stock_prices.py", line 83, in extract
    raise AirflowException(f"No time series data found for {symbol}. Response: {data}")
airflow.exceptions.AirflowException: No time series data found for GOOGL. Response: {'Information': 'Thank you for using Alpha Vantage! This is a premium endpoint. You may subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly unlock all premium endpoints'}
[2025-08-22T06:32:51.086+0000] {taskinstance.py:1206} INFO - Marking task as UP_FOR_RETRY. dag_id=stock_etl_pipeline, task_id=extract__2, run_id=scheduled__2025-08-21T00:00:00+00:00, execution_date=20250821T000000, start_date=20250822T063249, end_date=20250822T063251
[2025-08-22T06:32:51.096+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 6 for task extract__2 (No time series data found for GOOGL. Response: {'Information': 'Thank you for using Alpha Vantage! This is a premium endpoint. You may subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly unlock all premium endpoints'}; 243)
[2025-08-22T06:32:51.120+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-08-22T06:32:51.154+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-22T06:32:51.156+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
